rqt_gui_cpp/src/rqt_gui_cpp/nodelet_plugin_provider.hpp

```diff
@@ -30,48 +30,51 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef rqt_gui_cpp__NodeletPluginProvider_H
-#define rqt_gui_cpp__NodeletPluginProvider_H
+#ifndef RQT_GUI_CPP__NODELET_PLUGIN_PROVIDER_HPP_
+#define RQT_GUI_CPP__NODELET_PLUGIN_PROVIDER_HPP_
 
-#include <qt_gui_cpp/ros_pluginlib_plugin_provider.hpp>
+#include <unistd.h>
 
-#include <rqt_gui_cpp/plugin.h>
+#include <QThread>
+#include <iostream>
+#include <memory>
+#include <string>
 
-#include <rclcpp/rclcpp.hpp>
+#include <qt_gui_cpp/ros_pluginlib_plugin_provider.hpp>
 
-#include <QThread>
+#include <rqt_gui_cpp/plugin.hpp>
 
-#include <string>
-#include <unistd.h>
-#include <iostream>
+#include <rclcpp/rclcpp.hpp>
 
-namespace rqt_gui_cpp {
+namespace rqt_gui_cpp
+{
 
 class RosCppPluginProvider;
```

Comment:
I think we can remove this forward reference, and instead include the header file:

```suggestion
```

(and then up above add in `#include "./roscpp_plugin_provider.hpp"`